src/discordcr/mappings/guild.cr

```diff
@@ -117,6 +117,7 @@ module Discord
     JSON.mapping(
       id: {type: UInt64, converter: SnowflakeConverter},
       name: String,
+      permissions: UInt64,
```

Comment:
We should return a `Permissions` object here and every other time we have discord permissions.